Licensing_Program/config_handling.py

```diff
@@ -194,3 +203,11 @@ def load_configfile(cwd=".", info_level=""):
         )
 
     return config
+
+
+def genearate_default_configfile(cwd="."):
```

Comment:
Generate is misspelled. Also, since this function writes the config in addition to generating it, maybe change the name to `write_default_configfile(...)`.